schemas/Account.json

```diff
@@ -39,6 +39,11 @@
     "ledger": {
       "description": "A link the the account's ledger",
       "type": "Iri.json"
+    },
+    "fingerprint": {
+      "description": "A fingerprint of the account's client certificate",
+      "type": "string",
+      "pattern": "^[0-9A-F]{2}(:[0-9A-F]{2}){,127}$."
```

Comment:
The default digest used by OpenSSL for x509 certificates is SHA-1, which would make the fingerprint 40 characters. I can set that as a safe default minimum length